### PR TITLE
Removing ContextAttributes enums

### PR DESCRIPTION
**Pull request by slinkydeveloper**
They're useless and they're just increasing the size of the sdk

### CODE REVIEW

**Comment by pierDipi** [comment]
We're actually using them in `eventing-kafka-broker`, so they're not so useless.

**Comment by slinkydeveloper** [comment]
How are you using them? They're really necessary?

File: core/src/main/java/io/cloudevents/core/v03/CloudEventV03.java [hunk 1]
```diff
@@ -36,6 +36,16 @@
...
 
+    private final static String ID = "id";
+    private final static String SOURCE = "source";
+    private final static String SPECVERSION = "specversion";
+    private final static String TYPE = "type";
+    private final static String TIME = "time";
+    private final static String SCHEMAURL = "schemaurl";
+    private final static String DATACONTENTTYPE = "datacontenttype";
+    private final static String DATACONTENTENCODING = "datacontentencoding";
+    private final static String SUBJECT = "subject";
```
Comment (pierDipi) [inline core/src/main/java/io/cloudevents/core/v03/CloudEventV03.java:47]: Can you make those one public?

File: core/src/main/java/io/cloudevents/core/v1/CloudEventV1.java [hunk 2]
```diff
@@ -35,6 +35,15 @@
...
 public final class CloudEventV1 extends BaseCloudEvent {
 
+    private final static String ID = "id";
+    private final static String SOURCE = "source";
+    private final static String SPECVERSION = "specversion";
+    private final static String TYPE = "type";
+    private final static String TIME = "time";
+    private final static String DATASCHEMA = "dataschema";
+    private final static String DATACONTENTTYPE = "datacontenttype";
+    private final static String SUBJECT = "subject";
```
Comment (pierDipi) [inline core/src/main/java/io/cloudevents/core/v1/CloudEventV1.java:45]: Same here.

File: core/src/main/java/io/cloudevents/core/v03/CloudEventV03.java (hunk 1 above)
Comment (slinkydeveloper) [reply core/src/main/java/io/cloudevents/core/v03/CloudEventV03.java:47]: https://github.com/cloudevents/sdk-java/pull/282#issuecomment-729606022

Comment (pierDipi) [reply core/src/main/java/io/cloudevents/core/v03/CloudEventV03.java:47]: We can hardcode them or if you make those ones public we can use them.

**Comment by slinkydeveloper** [comment]
@Alfusainey @pierDipi wanna look again at this? I added javadocs + made constants public

**Review by pierDipi — APPROVED** [review]

**Pull request merged**